lib/rmt/scc_sync.rb

```diff
@@ -17,11 +17,17 @@ def sync
 
     @logger.info('Updating the database')
     data.each do |item|
-      @logger.debug("Adding product #{item[:name]}")
+      @logger.debug("Adding product #{item[:identifier]}/#{item[:version]}#{(item[:arch]) ? '/' + item[:arch] : ''}")
```

Comment:
Logger verbosity isn't configured in any way now.
Should we use the stock Ruby logger, or find a replacement for it, or maybe subclass it to modify output format?